ttkthemes/themed_tk.py

```diff
@@ -44,13 +44,21 @@ def __init__(self, *args, **kwargs):
         # Set initial theme
         if theme is not None and theme in self.get_themes():
             self.set_theme(theme, toplevel, background)
+        elif theme is None and background is not None:
+            self.set_background(background)
         self.__init__toplevel = tk.Toplevel.__init__
+        
+    def set_background(self, background):
+        self.config(background=background)
+        ttk.Style(self).configure(".", background=background)
 
     def set_theme(self, theme_name, toplevel=False, background=False):
         """Redirect the set_theme call to also set Tk background color"""
         ThemedWidget.set_theme(self, theme_name)
         color = ttk.Style(self).lookup("TFrame", "background", default="white")
-        if background is True:
+        if bool(background) is True and color == "#f5f6f7":
```

Comment:
I am not quite comfortable with this implementation. I would really much prefer to keep a `bool` type keyword argument, as I mention in #50 . Also, it is unconventional to hard-code colors or other values like this.